pdpserver/server.go

```diff
@@ -241,4 +241,7 @@ func (s *server) serve() {
 		// serveRequests() will be executed by external request.
 		log.Info("Waiting for policies to be applied.")
 	}
+	for {
+		time.Sleep(time.Second)
```

Comment:
Periodic sleeping is not a good way to wait forever. There are [bunch of better ways to do it](https://blog.sgmansfield.com/2016/06/how-to-block-forever-in-go/)